README.md

```diff
@@ -1,2 +1,87 @@
 # school-library
 building your school library app with Ruby programming language
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![screenshot]()
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+
+## Video Demo
+
+[Video Demo](https://www.loom.com/share/86a73ca9798443a2af325321d206d7ab)
```

Comment:
- [ ] This link leads to your Recipe app presentation from JS Capstone (I think). Kindly remove this section or add an appropriate video demo if you have one.